content/_help/fraud-concerns/fraud-concerns._es.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Preocupaciones de fraude 
+category: fraud-concerns
+permalink: /es/help/fraud-concerns/overview/
+order: 0
+---
+## ¿Le preocupa el fraude o la actividad no autorizada en su cuenta?
+
+Si sospecha que su cuenta de Login.gov se ha visto comprometida, o si observa actividad sospechosa en esta, estamos a su disposición para ayudarle a tomar medidas inmediatas para proteger su identidad y la seguridad de su cuenta.
+
+### Revise si hay actividad sospechosa
+Puede revisar los inicios de sesión recientes y los dispositivos asociados con su cuenta de Login.gov:
+
+{: .number-list}
+1. Inicie sesión en [Login.gov](http://secure.login.gov/es/).
+2. Navegue a la **página de su cuenta**.
+3. Seleccione **“Historial”** para ver los **dispositivos utilizados recientemente y los inicios de sesión**.
+
+Si ve algún dispositivo o inicio de sesión que no reconozca, le recomendamos encarecidamente que:
+
+   * **Cambie su contraseña de inmediato**, y
+   * Frases que solo usted conozca
```

Comment:
Something is off with this section in Spanish compared to English. Run the copy/text for a confirmation.